cla_public/static-src/javascripts/modules/textarea-character-count.js

```diff
@@ -27,13 +33,24 @@
       var $textArea = $(evt.target);
       var value = $textArea.val();
       var maxLength = $textArea.data('character-count');
-      var remainingCount = maxLength - value.length;
+      var numberOfLineReturns = (value.match(/\n/g) || []).length;
+      var remainingCount = maxLength - value.length - numberOfLineReturns;
```

Comment:
Just checked out the counter on staging via the `Contact` page. Seems that a new line is being counted as 2 characters instead of 1.

Reply:
That is because the 4000 character check at the back end counts each line return as 2 characters.  This is the cause of the bug where it was saying some characters were remaining yet it was failing validation.